File: daspro/tampil_arr.cpp
#include <stdio.h>

int main() {
    // List nama-nama buah (menggunakan array karakter 2D)
    char buah[4][10] = {"apel", "anggur", "mangga", "Jeruk"};
    int jumlahBuah = 4;  // Jumlah elemen dalam array

    // Ambil "mangga" dari indeks ke-2
    printf("Menampilkan salah satu isi array\n");
    printf("\nIsi buah indeks ke-2 adalah: %s\n", buah[2]);

    // Tampilkan semua daftar buah
    printf("\nMenampilkan semua isi array:\n");
    printf("\nSemua Buah: ada %d macam\n", jumlahBuah);
    for (int i = 0; i < jumlahBuah; i++) {
        printf("%s\n", buah[i]);
    }

    return 0;
}

File: daspro/tampil_arr.pas
program DaftarBuah;
uses crt;
var
    buah: array[1..4] of string[10];
    i: integer;
begin
    clrscr;
    // List nama-nama buah
    buah[1] := 'apel';
    buah[2] := 'anggur';
    buah[3] := 'mangga';
    buah[4] := 'Jeruk';

    // Ambil "mangga" dari indeks ke-3
    writeln('Menampilkan salah satu isi array');
    writeln('Isi buah index ke-3 adalah: ', buah[3]);
    writeln;

    // Tampilkan semua daftar buah
    writeln('Menampilkan semua isi array:');
    writeln('Semua Buah: ada ', 4, ' macam');
    for i := 1 to 4 do
    begin
        writeln(buah[i]);
    end;
    readln;
end.
